src/Accelerators/NNPA/Conversion/ONNXToZHigh/PerfModelArch15.inc

```diff
@@ -0,0 +1,349 @@
+         9.546897460120308e-10 * complexity2;
+}
+
+// Operation Stick_3ds: estimated times.
+double arch15_estimatedTimeForCPU_Stick_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * e2 * e1;
+  // Regression for CPU with r2 = 0.9573201775683942
+  return 7.223242630385479e-07 + 8.501740539965986e-11 * complexity;
+}
+// Operation Stick_3ds: estimated times.
+double arch15_estimatedTimeForNNPA_Stick_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * ms_ceiling(e2, 2.0) * ms_ceiling(e1, 64.0);
+  double complexity2 = e3 * ms_ceiling(e2, 32.0) * ms_ceiling(e1, 64.0);
+  // Regression for NNPA with r2 = 0.6906958856336693
+  return 1.4650992868656998e-06 + 1.5980625370318473e-10 * complexity +
+         1.5980625370318473e-10 * complexity2;
+}
+
+// Operation Sub_3ds: estimated times.
+double arch15_estimatedTimeForCPU_Sub_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * e2 * e1;
+  // Regression for CPU with r2 = 0.9994300646660222
+  return 8.53949178130232e-07 + 9.552545176353334e-11 * complexity;
+}
+// Operation Sub_3ds: estimated times.
+double arch15_estimatedTimeForNNPA_Sub_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * ms_ceiling(e2, 2.0) * ms_ceiling(e1, 64.0);
+  double complexity2 = e3 * ms_ceiling(e2, 32.0) * ms_ceiling(e1, 64.0);
+  // Regression for NNPA with r2 = 0.9953310347130198
+  return 2.3492432002936522e-06 + 1.723261703570422e-11 * complexity +
+         1.723261703570422e-11 * complexity2;
+}
+
+// Operation Tanh_3ds: estimated times.
+double arch15_estimatedTimeForCPU_Tanh_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * e2 * e1;
+  // Regression for CPU with r2 = 0.9992625489601716
+  return 3.2369598896957175e-07 + 1.4078506521178395e-09 * complexity;
+}
+// Operation Tanh_3ds: estimated times.
+double arch15_estimatedTimeForNNPA_Tanh_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * ms_ceiling(e2, 2.0) * ms_ceiling(e1, 64.0);
+  double complexity2 = e3 * ms_ceiling(e2, 32.0) * ms_ceiling(e1, 64.0);
+  // Regression for NNPA with r2 = 0.9960934032072877
+  return 1.9722513558089736e-06 + 1.790114983535808e-11 * complexity +
+         1.7901149835358073e-11 * complexity2;
+}
+
+// Operation Unstick_3ds: estimated times.
+double arch15_estimatedTimeForCPU_Unstick_3ds(double e3, double e2, double e1) {
+  double complexity = e3 * e2 * e1;
+  // Regression for CPU with r2 = 0.9932274704371545
+  return 7.653934223015085e-07 + 1.389382010359011e-10 * complexity;
+}
+// Operation Unstick_3ds: estimated times.
+double arch15_estimatedTimeForNNPA_Unstick_3ds(
+    double e3, double e2, double e1) {
+  double complexity = e3 * ms_ceiling(e2, 2.0) * ms_ceiling(e1, 64.0);
+  double complexity2 = e3 * ms_ceiling(e2, 32.0) * ms_ceiling(e1, 64.0);
+  // Regression for NNPA with r2 = 0.4294645871109005
+  return 1.0784429022719318e-05 + 1.569351314487559e-10 * complexity +
+         1.5693513144875584e-10 * complexity2;
+}
```

Comment:
If the weight constants are stored in a file, will it be easier to change the value for different arch, op and system?